### PR TITLE
Op executesc add datastore

### PR DESCRIPTION
**Pull request by sydhds**

### CODE REVIEW

File: massa-models/src/operation.rs [hunk 1]
```diff
@@ -695,14 +722,18 @@ impl Deserializer<OperationType> for OperationTypeDeserializer {
                         context("Failed data deserialization", |input| {
                             self.data_deserializer.deserialize(input)
                         }),
+                        context("Failed data deserialization", |input| {
```
Comment (AurelienFT) [inline massa-models/src/operation.rs:725]: The message here is the same as above. It should be an other to be able to find easily the source of the error message.

**Review by AurelienFT — COMMENTED** [review]
Very good usage of serialization/deserialization.

File: massa-models/src/operation.rs (hunk 1 above)
Comment (sydhds) [reply massa-models/src/operation.rs]: Just fixed the error message :)

**Review by damip — COMMENTED** [review]
Very good work !

**Pull request closed without merging**